example/spec/parallel/discoverGranules/DiscoverGranulesHttpsSpec.js

```diff
@@ -108,6 +108,7 @@ xdescribe('The Discover Granules workflow with https Protocol', () => {
       expect(lambdaOutput.payload.granules.length).toEqual(3);
       expect(lambdaOutput.payload.granules[0].granuleId).toEqual('granule-1');
       expect(lambdaOutput.payload.granules[0].files.length).toEqual(2);
+      expect(lambdaOutput.payload.granules[0].files[0].fileType).toEqual('data');
```

Comment:
See https://github.com/nasa/cumulus/pull/867/files#r264986915 .  Again, no point in setting/checking checksum filetype here.